clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -2213,12 +2210,12 @@ bool tools::GetSDLFromOffloadArchive(
 
 // Wrapper function used by driver for adding SDLs during link phase.
 void tools::AddStaticDeviceLibsLinking(Compilation &C, const Tool &T,
-                                const JobAction &JA,
```

Comment:
Drop the indentation change. Fixed by c1afed9f48c2d4ebb21ac1672948e82d23c69174 2 days ago.

Reply:
Thanks.  I rebased my patch.